core/src/main/scala/org/apache/spark/api/java/JavaPairRDD.scala

```diff
@@ -234,6 +234,9 @@ class JavaPairRDD[K, V](val rdd: RDD[(K, V)])
    * In addition, users can control the partitioning of the output RDD, the serializer that is use
    * for the shuffle, and whether to perform map-side aggregation (if a mapper can produce multiple
    * items with the same key).
+   *
+   * @note V and C can be different -- for example, one might group a RDD of type (Int, Int) into
+   * an RDD of type (Int, List[Int]).
```

Comment:
It seems fine now.

![2016-11-18 4 47 33](https://cloud.githubusercontent.com/assets/6477701/20422606/c57c1618-adae-11e6-8d65-b6f15a9ddb3a.png)
![2016-11-18 4 47 17](https://cloud.githubusercontent.com/assets/6477701/20422605/c579f63a-adae-11e6-98ca-b072924be75a.png)